packages/gatsby-plugin-page-creator/gatsby-node.js

```diff
@@ -0,0 +1,71 @@
+const globCB = require(`glob`)
+const Promise = require(`bluebird`)
+const _ = require(`lodash`)
+const chokidar = require(`chokidar`)
+const systemPath = require(`path`)
+
+const glob = Promise.promisify(globCB)
+
+const createPath = require(`./create-path`)
+const validatePath = require(`./validate-path`)
+
+// Path creator.
+// Auto-create pages.
+// algorithm is glob /pages directory for js/jsx/cjsx files *not*
+// underscored. Then create url w/ our path algorithm *unless* user
+// takes control of that page component in gatsby-node.
+exports.createPagesStatefully = async (
+  { store, boundActionCreators },
+  pluginOptions,
+  doneCb
+) => {
+  const { createPage, deletePage } = boundActionCreators
+  const program = store.getState().program
+  const exts = program.extensions.map(e => `${e.slice(1)}`).join(`,`)
+  const pagesDirectory = systemPath.posix.join(program.directory, pluginOptions.path)
```

Comment:
The change occurred on this line.
Before:
```
const pagesDirectory = systemPath.posix.join(program.directory, 'src/pages')
```

After
```
const pagesDirectory = systemPath.posix.join(program.directory, pluginOptions.path)
```

Reply:
If you don't see anywhere else where `src/pages` is hard-coded than no.